rocketpy/mathutils/function.py

```diff
@@ -1420,47 +1419,42 @@ def __interpolate_polynomial__(self):
             )
             return self.set_interpolation("spline")
         # Create coefficient matrix1
-        A = np.zeros((degree + 1, degree + 1))
+        matrix = np.zeros((degree + 1, degree + 1))
         for i in range(degree + 1):
-            A[:, i] = x**i
+            matrix[:, i] = x**i
         # Solve the system and store the resultant coefficients
-        self.__polynomial_coefficients__ = np.linalg.solve(A, y)
+        self.__polynomial_coefficients__ = np.linalg.solve(matrix, y)
```

Comment:
I know PyLint flags this, but between `matrix` and `A` I almost prefer `A` since it is a common "standard" in mathematical interpolation equation. However, perhaps an even better name would be: `sys_coefficient` or `vandermond_matrix`.

See: https://en.wikipedia.org/wiki/Polynomial_interpolation#Second_proof

Reply:
Honestly, I'm not a fan of the idea of violating the [style guide](https://docs.rocketpy.org/en/latest/development/style_guide.html#code-style) using "mathematical excuses" as this is not a final attribute.
I can quickly pass my eyes through this function and reading `A` is harder than the complete form `matrix_A`.

Given your link, it seems like vandermond_matrix is another thing (the X), so I think we could stay with `sys_coeffs`.

This variable occurs only 3 times in the code, not the end of the world.

I solved the problem here: [3fa58a1](https://github.com/RocketPy-Team/RocketPy/pull/446/commits/3fa58a1831229c8084b99581a0c1099017aa5cf4)
Open for future discussions.